docs/using-the-sdk/README.md

```diff
@@ -6,4 +6,4 @@ parent:
 # Using the SDK
 
 - [Modules](../../x/README.md)
-- [Simulation](./simulation.md)
+- [Simulation](../building-modules/simulator.md)
```

Comment:
This works although I'm wondering if we should point to `../core/simulation.md` instead, which introduces the simulator at a high-level and then provides a link at the bottom to `../building-modules/simulator.md`, which focuses on building simulator functionality into sdk modules. Thoughts?

Reply:
That seems good to me

Reply:
```suggestion
- [Simulation](../core/simulation.md)
```